README.md

```diff
@@ -1,193 +1,15 @@
-# Cloudformation
+# OPA CloudFormation Hook
```

Comment:
We call this "OPA AWS CloudFormation Hook" in the tutorial, and it's also the name of the repo.. so I'd suggest we go with that here too.